### ISSUE TITLE
Allow CORS in vachan-api V2

### ISSUE THREAD

**Issue by RevantCI** [Bridgeconn/vachan-api#134]
Allow CORS in vachan-api V2 to allow frontend to access the api when frontend is deployed from the localhost or other servers.
See https://fastapi.tiangolo.com/tutorial/cors/ for more details.
For 

> allow_origins - A list of origins that should be permitted to make cross-origin requests. E.g. ['https://example.org', 'https://www.example.org']. You can use ['*'] to allow any origin

Suggest to use  ['*'] to allow any origin for now

**Comment by kavitharaju** [comment]
Fixes this temporarily in #135 .
But it needs to be dealt properly while adding Auth

**Comment by kavitharaju** [comment]
@joelthe1 
[This](https://github.com/Bridgeconn/vachan-api/blob/ce59b939c6293bf88c195b966b97e75a2278834b/app/main.py#L29) is the CORS setting now. 
```

app.add_middleware(
    CORSMiddleware,
    allow_origins=["*"],
    # allow_credentials=True,
    allow_methods=["*"],
    allow_headers=["*"],
)
```
We are implementing auth based on user registered via Kratos and access rights based on user roles, requesting app's name provided in header etc.
Any changes required in this CORS setting?
